Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG GOLANG_BUILDER=golang:1.19
-ARG OPERATOR_BASE_IMAGE=gcr.io/distroless/static:nonroot
+ARG GOLANG_BUILDER=golang:1.20
```

Comment:
If we're bumping the Goland version here, I think we'll want to do it in the `go.mod`s as well